tests/wpt/tests/html/semantics/embedded-content/the-iframe-element/srcdoc-removed-iframe-crash.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Let's add a title element like "iframe with srcdoc content that loads after iframe is removed from the document".